packages/bpk-component-image/src/withLazyLoading.tsx

```diff
@@ -130,7 +114,9 @@ export default function withLazyLoading(
             return supportsPassiveOption;
           },
         });
+        // @ts-expect-error TODO: make this TS compliant
         window.addEventListener('test', null, opts);
+        // @ts-expect-error TODO: make this TS compliant
```

Comment:
changing this causes the unit tests to fail so it needs a bit of a bigger investigation. Added a ts exception for now

Reply:
Same here